solidity/random-beacon/test/utils/groups.ts

```diff
@@ -13,14 +13,19 @@ export async function createGroup(
   signers: Operator[]
 ): Promise<void> {
   const { blockNumber: startBlock } = await randomBeacon.genesis()
+  const submitterIndex = 1
   await mineBlocks(constants.offchainDkgTime)
   await signAndSubmitDkgResult(
     randomBeacon,
     blsData.groupPubKey,
     signers,
     startBlock,
-    noMisbehaved
+    noMisbehaved,
+    submitterIndex
   )
+
   await mineBlocks(params.dkgResultChallengePeriodLength)
-  await randomBeacon.approveDkgResult()
+  await randomBeacon
+    .connect(await ethers.getSigner(signers[submitterIndex - 1].address))
```

Comment:
Not in this PR but thoughts on adding signer to `Operator` struct:
https://github.com/keep-network/keep-core/blob/fcfb88fd1352e9defd549c21b2809cba4d98ee29/solidity/random-beacon/test/utils/sortitionpool.ts#L9
?
cc @nkuba 

I see we often have to use `ethers.getSigner` in our tests.

Reply:
I agree, we could add `signer` to the `Operator` structure.
Should we work on it now, or add it to the backlog, so we can do this once we have all the solidity tasks done?

Reply:
Let's backlog it, it's definitely nothing on the top of our priority list.

Reply:
Done